tests/api/reports-import.php

```diff
@@ -134,7 +136,7 @@ public function test_import_params() {
 		$reports  = $response->get_data();
 
 		$this->assertEquals( 200, $response->get_status() );
-		$this->assertCount( 1, $reports );
+		$this->assertCount( 2, $reports );
```

Comment:
Two orders were created, so the `$reports` would have two items. I have no idea how this passed before.